proto/ibc/core/client/v1/query.proto

```diff
@@ -148,11 +155,13 @@ message QueryClientParamsResponse {
   // params defines the parameters of the module.
   Params params = 1;
 }
+
 // QueryUpgradedClientStateRequest is the request type for the
 // Query/UpgradedClientState RPC method
 message QueryUpgradedClientStateRequest {
-  // client state unique identifier
-  string client_id = 1;
+  // client id field has been deprecated.
+  reserved 1;
+  reserved "option";
   // plan height of the current chain must be sent in request
   // as this is the height under which upgraded client state is stored
   int64 plan_height = 2;
```

Comment:
we should deprecate this as well imo. 

There should only ever exist 1 upgrade plan at a time. It might be an IBC upgrade and it might not, we should return this info in the response rather than take it as an arg in the request